src/tests/stdlib/change_owner.ab

```diff
@@ -12,5 +12,5 @@ main {
     } else {
         echo "File {tmpdir}/amber-symbolic not exists"
     }
-    trust $ rm -fr {tmpdir} $
+    trust $ rm -rf {tmpdir} $
```

Comment:
how do you pronounce it? remove rf tmpdir? why not "remove for real the tmpdir"?

Reply:
They're equivalent, but I've always seen it written `-rf` for `--recursive --force`. And a quick Google search turned up way more hits for that than for `-fr`.